src/psyclone/psyir/transformations/value_range_check.py

```diff
@@ -42,38 +42,36 @@
    actual code.
 '''
 
-from __future__ import absolute_import
-
 from psyclone import psyGen
-from psyclone.psyir.nodes import NanTestNode
+from psyclone.psyir.nodes import ValueRangeCheckNode
 from psyclone.psyir import nodes
 from psyclone.psyir.transformations.read_only_verify_trans \
     import ReadOnlyVerifyTrans
 
 
-class NanTestTrans(ReadOnlyVerifyTrans):
-    '''This transformation inserts a NanTestNode into the PSyIR of a
+class ValueRangeCheck(ReadOnlyVerifyTrans):
```

Comment:
By convention, this should be `ValueRangeCheckTrans` and the file re-named accordingly (plus the test file).

Reply:
Oh yes, that was a pain :( But done.

Reply:
Thanks.